src/phoenix/trace/otel/utils.py

```diff
@@ -0,0 +1,115 @@
+import json
+from datetime import datetime, timezone
+from typing import Any, Container, Iterable, Iterator, Mapping, Sequence, Tuple
+
+from opentelemetry.proto.common.v1.common_pb2 import AnyValue, ArrayValue, KeyValue, KeyValueList
+from opentelemetry.util.types import Attributes, AttributeValue
+from typing_extensions import assert_never
+
+from phoenix.trace.semantic_conventions import (
+    DOCUMENT_CONTENT,
+    DOCUMENT_ID,
+    DOCUMENT_METADATA,
+    DOCUMENT_SCORE,
+    EMBEDDING_TEXT,
+    EMBEDDING_VECTOR,
+)
+
+
+def decode_document(key_values: Iterable[KeyValue]) -> Iterator[Tuple[str, Any]]:
```

Comment:
Nit: Looks like most of the functions in this file are only used in one of `encode.py` or `decode.py`. Are we anticipating they'll be used elsewhere? Otherwise, might be nice to keep it all together.